samples/nrf9160/gps/Kconfig

```
@@ -17,12 +17,32 @@ choice GPS_SAMPLE_ANTENNA
 
 config GPS_SAMPLE_ANTENNA_ONBOARD
 	bool "Use onboard antenna"
+	depends on !BOARD_CIRCUITDOJO_FEATHER_NRF9160NS
 
 config GPS_SAMPLE_ANTENNA_EXTERNAL
 	bool "Use external antenna"
 
 endchoice
 
+config GPS_SAMPLE_AT_MAGPIO
+	string "AT%MAGPIO command"
+	default "AT\%XMAGPIO=1,0,0,1,1,1574,1577" if BOARD_NRF9160DK_NRF9160NS
+	default "AT\%XMAGPIO=1,1,1,7,1,746,803,2,698,748,2,1710,2200,3,824,894,4,880,960,5,791,849,7,1565,1586" if BOARD_THINGY91_NRF9160NS
+	help
+	  Defines what is the AT%MAGPIO command to be sent to GPS module. Leave
+	  empty if this commands should not be sent.
+
+config GPS_SAMPLE_AT_COEX0
+	string "AT%COEX0 command"
+	default "AT\%XCOEX0=1,1,1565,1586" if (BOARD_NRF9160DK_NRF9160NS && GPS_SAMPLE_ANTENNA_ONBOARD)
+	default "AT\%XCOEX0" if (BOARD_NRF9160DK_NRF9160NS && GPS_SAMPLE_ANTENNA_EXTERNAL)
+	default "AT\%XCOEX0=1,1,1565,1586" if (BOARD_THINGY91_NRF9160NS && GPS_SAMPLE_ANTENNA_ONBOARD)
+	default "AT\%XCOEX0" if (BOARD_THINGY91_NRF9160NS && GPS_SAMPLE_ANTENNA_EXTERNAL)
```

DK and Thingy configuration could be combined in the same way as in the GPS driver Kconfig:
```suggestion
	default "AT\%XCOEX0=1,1,1565,1586" if (BOARD_NRF9160DK_NRF9160NS || BOARD_THINGY91_NRF9160NS) && GPS_SAMPLE_ANTENNA_ONBOARD
	default "AT\%XCOEX0" if (BOARD_NRF9160DK_NRF9160NS || BOARD_THINGY91_NRF9160NS) && GPS_SAMPLE_ANTENNA_EXTERNAL
```